jenkins.sh

```diff
@@ -10,5 +10,7 @@ pip2.7 install sphinx
 make develop extras=[aws,mesos,azure,encryption,cwl]
 export LIBPROCESS_IP=127.0.0.1
 export PYTEST_ADDOPTS="--junitxml=test-report.xml"
+rm -rf /mnt/ephemeral/tmp
```

Comment:
Doesn't this fail if the directory doesn't exist? The way jenkins.sh is run is that any failed command leads to build failure.